src/Services/QueuedJobService.php

```diff
@@ -66,6 +66,37 @@ class QueuedJobService
      */
     private static $stall_threshold = 3;
 
+    /**
+     * How early stuck jobs will become eligible for automated retry processing (minutes)
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_sentinel = 1;
+
+    /**
+     * How many jobs are eligible to be automatically retried per one health check
+     * Setting this to 0 disables the job retry feature
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_limit = 0;
+
+    /**
+     * Defines the job status transformation for job retries
+     * Set rule value to "null" to disable it
```

Comment:
```suggestion
     * Map of stuck job statuses to new status when retrying the job. For example a "broken" job should be turned into a "new" job for it to be retried correctly.
     * Set the value to "null" for any given stuck job status to disable retries for that status.
```
I've reworded this to make it a bit clearer how this map works, based on my understanding of it.